.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 db/config/config.json
 package-lock.json
 *.bk
+.idea
```

Comment:
Though you have added the idea folder to the ignore, you have already committed the contents of the folder, so it will persist. You need to [manually remove](https://gettinggit.com/learn/master-git-rm) the .idea dir from your repo/pr.